microsoft/vscode
# Sticky Scroll : when several outline models are available, choose the one spanning the biggest total range

In particular it fixes https://github.com/microsoft/vscode/issues/159136

![sticky-scroll-multiple-providers](https://user-images.githubusercontent.com/61460952/186920385-51ba7c73-ef07-411a-8f3c-66c6a36e3546.gif)

To test:

Open a launch.json file
Enable sticky scroll and verify that the sticky lines stuck to the top of the view port do not correspond to the outline model in the bottom-right pannel
